pkg/channel/consolidated/reconciler/controller/resources/dispatcher.go

```diff
@@ -56,8 +58,9 @@ func MakeDispatcher(args DispatcherArgs) *v1.Deployment {
 			Kind:       "Deployments",
 		},
 		ObjectMeta: metav1.ObjectMeta{
-			Name:      dispatcherName,
-			Namespace: args.DispatcherNamespace,
+			Name:            dispatcherName,
+			Namespace:       args.DispatcherNamespace,
+			OwnerReferences: []metav1.OwnerReference{args.OwnerRef},
```

Comment:
Should we add the `KafkaChannel` itself to the `Args`, and do smth like:
```go
OwnerReferences: []metav1.OwnerReference{*kmeta.NewControllerRef(args.kc)},
```
we do that w/ the source, like here: https://github.com/knative-sandbox/eventing-kafka/blob/83e1b7a7c4353f129e83e39f137f4f662765d242/pkg/source/reconciler/source/resources/receive_adapter.go#L86

Reply:
that would mean you're setting the ownerRef of the dispatcher deployment to be owned by that KafkaChannel, which is not what we want to do here. This true in the source case because the receive adapter is per KafkaSource, but here the dispatcher is shared across all KafkaChannel object.